bat-utils/lib/extras-utils.js

```diff
@@ -48,6 +49,11 @@ function documentOlderThan (olderThanDays, anchorTime, _id) {
   return createdTimestamp(_id) < (anchorTime - (DAY_MS * olderThanDays))
 }
 
+function hexTime (now = new Date()) {
```

Comment:
I would prefer we call this something like `timestampToMongoId` or similar

Reply:
can do

Reply:
just realized it's not even used anymore, so removing